pages/linux/abbr.md

```diff
@@ -0,0 +1,21 @@
+# abbr
+
+> Manages abbreviations for the fish shell.  
+> User-defined words are replaced with longer phrases after they are entered.  
+> More information: <https://fishshell.com/docs/current/cmds/abbr.html>.
+
+- Add a new abbreviation:
+
+`abbr -a -g gco git checkout`
```

Comment:
It's better to use long options to make tldr pages more readable and memorizable:

```suggestion
`abbr --add gco git checkout`
```

Maybe better to remove `-g` flag to make abbreviation universal and automatically visible in other shell sessions.